### PR TITLE
split example into smaller examples

### PR DESCRIPTION
**Pull request by karthik2804**
Split the example into multiple smaller examples.

### CODE REVIEW

File: examples/hello_world/Pipfile
Comment (dicej) [inline examples/hello_world/Pipfile:4]: You could probably just delete this file (and Pipfile.lock) entirely if there are no dependencies.

File: examples/outbound_http/README.md
Comment (dicej) [inline examples/outbound_http/README.md:3]: Grammar nit:
```suggestion
A simple example showcasing an outbound HTTP request in Spin Python SDK.
```

File: examples/outbound_redis/app.py
Comment (dicej) [inline examples/outbound_redis/app.py:6]: Optional: could show off `execute` here, e.g.
```python
    redis_execute("redis://127.0.0.1:6379", "set", [b"foo", b"hello"])
    redis_execute("redis://127.0.0.1:6379", "append", [b"foo", b", world!"])
```

File: examples/outbound_redis/app.py
Comment (dicej) [inline examples/outbound_redis/app.py:22]: I'd suggest leaving the `request.uri` out, since it has nothing to do with the rest of the sentence.
```suggestion
                    bytes(f"Executed outbound Redis commands", "utf-8"))
```

**Comment by karthik2804** [comment]
I have resolved all the mentioned suggestions above. I will wait until the KV PR is merged to add a KV example and also potentially an example of using external packages.

**Comment by karthik2804** [comment]
The resolved changes and additional examples for KV and external library usage have now been pushed.

**Comment by tpmccallum** [comment]
Hi @karthik2804 
Not sure if this is intentional (or to do with this specific PR) but I noticed that the outbound request example returns an additional `, the environment: environ({})` and the end of the response. 

```bash
Here is a dog fact: {"fact":"Dogs could participate in doggie blood drives: dogs are able to donate blood to fellow pooches."}, the environment: environ({})%  
```

**Comment by karthik2804** [comment]
@tpmccallum Thanks for catching that, it has been fixed now.

**Review by dicej — APPROVED** [review]

**Pull request merged**
